khelo/lib/ui/flow/score_board/components/select_wicket_taker_sheet.dart

```diff
@@ -41,88 +41,50 @@ class _SelectWicketTakerSheetState
+        )
+      ],
     );
   }
 
-  Widget _body(BuildContext context, ScoreBoardViewState state) {
-    return _specificPlayerSelectionView(context, state);
-  }
-
-  Widget _sectionTitle(BuildContext context, String title) {
+  Widget _wicketTakerContent(BuildContext context, ScoreBoardViewState state) {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        const SizedBox(
-          height: 24,
-        ),
         Text(
-          title,
-          style: AppTextStyle.header1
-              .copyWith(color: context.colorScheme.textSecondary),
+          context.l10n.score_board_choose_fielder_title,
+          style: AppTextStyle.header3
+              .copyWith(color: context.colorScheme.textPrimary),
         ),
         const SizedBox(
           height: 16,
         ),
-      ],
-    );
-  }
-
-  Widget _profilePlaceHolder(BuildContext context, {required double size}) {
-    return Container(
-      height: size,
-      width: size,
-      decoration: BoxDecoration(
-        color: context.colorScheme.containerLow,
-        shape: BoxShape.circle,
-      ),
-      child: const Icon(Icons.person),
-    );
-  }
-
-  Widget _specificPlayerSelectionView(
-      BuildContext context, ScoreBoardViewState state) {
-    final list = getFilteredList(state);
-    return Stack(
-      children: [
-        ListView(
+        Wrap(
+          spacing: 16,
+          runSpacing: 16,
           children: [
-            _sectionTitle(
-                context, context.l10n.score_board_choose_fielder_title),
-            GridView.builder(
-              physics: const NeverScrollableScrollPhysics(),
-              shrinkWrap: true,
-              gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
-                  crossAxisCount: 3,
-                  crossAxisSpacing: 16,
-                  mainAxisSpacing: 16,
-                  childAspectRatio: 0.7),
-              itemCount: list.length,
-              itemBuilder: (context, index) {
-                return _userCell(
-                  context: context,
-                  user: list[index].player,
-                  isSelected: selectedId == list[index].player.id,
-                  onTap: () {
-                    setState(() {
-                      selectedId = list[index].player.id;
-                    });
-                  },
-                );
-              },
-            )
+            for (final player in getFilteredList(state)) ...[
+              UserCellView(
+                title:
+                    player.player.name ?? context.l10n.common_anonymous_title,
+                imageUrl: player.player.profile_img_url,
+                initial: player.player.nameInitial,
+                isSelected: selectedId == player.player.id,
+                onTap: () {
+                  setState(() {
+                    selectedId = player.player.id;
+                  });
+                },
+              )
+            ],
```

Comment:
Can you use map and if this wrap widget children only this list so define direct like this

```suggestion
            children: getFilteredList(state)
              .map(
                (player) => UserCellView(
                  title:
                      player.player.name ?? context.l10n.common_anonymous_title,
                  imageUrl: player.player.profile_img_url,
                  initial: player.player.nameInitial,
                  isSelected: selectedId == player.player.id,
                  onTap: () {
                    setState(() {
                      selectedId = player.player.id;
                    });
                  },
                ),
              )
              .toList(),
```